examples/next-prisma-starter-websockets/src/server/context.ts

```diff
@@ -6,28 +6,20 @@ import { IncomingMessage } from 'http';
 import { getSession } from 'next-auth/react';
 import ws from 'ws';
 
-const prisma = new PrismaClient({
-  log:
-    process.env.NODE_ENV === 'development'
-      ? ['query', 'error', 'warn']
-      : ['error'],
-});
 /**
  * Creates context for an incoming request
  * @link https://trpc.io/docs/context
  */
-export const createContext = async ({
-  req,
-  res,
-}:
-  | trpcNext.CreateNextContextOptions
-  | NodeHTTPCreateContextFnOptions<IncomingMessage, ws>) => {
-  const session = await getSession({ req });
+export const createContext = async (
+  opts:
+    | trpcNext.CreateNextContextOptions
+    | NodeHTTPCreateContextFnOptions<IncomingMessage, ws>,
+) => {
+  const session = await getSession(opts);
+
   console.log('createContext for', session?.user?.name ?? 'unknown user');
+
   return {
-    req,
-    res,
-    prisma,
```

Comment:
Why are we not sending the prisma client through context anymore?

Reply:
Unnecessary abstraction. Prisma is global and there's no functionality to bind contextual information to it AFAIK. 

Related: https://github.com/prisma/prisma/issues/7596